cmd/backup-manager/app/import/restore.go

```diff
@@ -58,14 +58,16 @@ func (ro *Options) loadTidbClusterData(restorePath string) error {
 		return fmt.Errorf("dir %s does not exist or is not a dir", restorePath)
 	}
 	args := []string{
-		fmt.Sprintf("-d=%s", restorePath),
-		fmt.Sprintf("-h=%s", ro.Host),
-		fmt.Sprintf("-P=%d", ro.Port),
-		fmt.Sprintf("-u=%s", ro.User),
-		fmt.Sprintf("-p=%s", ro.Password),
+		"--status-addr=0.0.0.0:8289",
```

Comment:
We may have to specify the log-file to "" so that the logs of lightning can be captured in L72.